packages/__mocks__/gatsby-core-utils.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    isCI: () => false,
```

Comment:
Not sure if this file should be in this directory since the `packages` directory typically only has package folders?

Reply:
Shall I move it to the root of the repo then? Otherwise, without mocking it, when I run 
```
yarn jest is-tty.js
```
for example, I was getting an error that the module could not be found.

But I think I found the problem: if I don't build the packages with `yarn lerna-prepare`, then I get this kind of errors and they disappeared afterwards.

I'll remove the mock file entirely!

Reply:
Good idea!